CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Change log
 
+## 2.2.3 (2021-12-27)
```

Comment:
All the new changes go under the `master` section. PRs cannot bump versions (because you don't know when it's gonna be merged).